lib/wasm/WebAssemblyGenerator.js

```diff
@@ -337,9 +327,9 @@ const addInitFunction = ({
 		funcBody.push(t.callInstruction(startAtFuncIndex));
 	}
 
-	for (const instr of additionalInitCode) {
-		funcBody.push(instr);
-	}
+	// Add additional code and params
+	funcBody.push(...additionalInitCode);
```

Comment:
I intentionally avoid `push(...array)` because it crashes for big arrays.

``` js
> array.length
131072
> var a = [];
undefined
> a.push(...array)
RangeError: Maximum call stack size exceeded
> for(const i of array) a.push(i);
131072
```

Reply:
Ah I was wondering why, ok thanks.